src/plugins/controls/timer/component/countdown.js

```diff
@@ -117,8 +120,10 @@ export default function countdownFactory($container, config) {
                         encodedTime = timeEncoder.encode(this.remainingTime / precision);
                         if (encodedTime !== this.encodedTime) {
                             this.encodedTime = encodedTime;
+                            [hours, minutes/* , seconds */] = this.encodedTime.split(':');
```

Comment:
Is it intended to keep dead code? You should remove it.
Why mixing ES6 and ES5?